tests/unit/adapters/drf-test.js

```diff
@@ -33,6 +33,7 @@ test('buildURL', function(assert) {
   var adapter = this.subject();
   assert.equal(adapter.buildURL('Animal', 5, null), 'test-host/test-api/animals/5/');
   assert.equal(adapter.buildURL('FurryAnimals', 5, null), 'test-host/test-api/furry-animals/5/');
+  assert.equal(adapter.buildURL('Animal', null, null, 'query', { limit: 10 }), 'test-host/test-api/animals/');
```

Comment:
ah ok, that's a little bizarre that they would have a urlForQuery method but not actually add the query string until the ajax call.  but it looks like we should be ok with this behavior.  we've been using it already with things like coalesceFindRequests and whatnot and it seems to be working.  i guess this is for some backend that wants queries built straight into the URL or something.